### ISSUE TITLE
IdTokenHint issue with IdentityServer4 and AAD 

### ISSUE THREAD

**Issue by Oleksandr-Tokmakov** [IdentityServer/IdentityServer4#4866]
Using IdentityModel client I generate LogoutRequest to endsession endpoint with id token hint parameter:
```
LogoutRequest logoutRequest = new LogoutRequest
{
    IdTokenHint = user.IdentityToken
};
string logoutUrl = await client.PrepareLogoutAsync(logoutRequest, cancellation.Token);
```
It works as expected, but after this step IdentityServer4 Quickstart sample redirects browser to \account\logout?logoutId=...
I'm okay with it but at this step I don't see idtokenhint parameter and in 

`public async Task<IActionResult> Logout(LogoutInputModel model)`

It redirects to AAD logout page but due to missing idtokenhint logout page prompts user to select account.
I'd like to skip this prompt. And as I understood I need to have idtokenhint. 
Also, in this method it I have added SignOut call to signInManager:
```
if (User?.Identity.IsAuthenticated == true)
            {
                // delete local authentication cookie
                await HttpContext.SignOutAsync();

                **await signInManager.SignOutAsync();**

                // raise the logout event
                await _events.RaiseAsync(new UserLogoutSuccessEvent(User.GetSubjectId(), User.GetDisplayName()));
            }
```
Because without it keeps redirecting user back to aad logout page.

It looks like I do something wrong. 

My main questions are:
 - how quicksample redirects from endsession to account\logout endpoint
 - where idtokenhint disappears?

**Comment by leastprivilege** [comment]
This might be related to a bug we fixed for the upcoming 4.1.

**Comment by Oleksandr-Tokmakov** [comment]
> This might be related to a bug we fixed for the upcoming 4.1.

Can I get somewhere this version?
